fedora/templates/csv/timers_enabled.csv

```diff
@@ -0,0 +1 @@
+dnf-automatic,dnf-automatic
```

Comment:
Can you add the documentation header similar to other templates? It is hard to know which format it accepts and which parameters are optional.